sql/mob_pools.sql

```diff
@@ -5992,26 +5992,26 @@ INSERT INTO `mob_pools` VALUES (5932,'fablinix','Fablinix',327,0x0000DC0B0000000
 INSERT INTO `mob_pools` VALUES (5937,'cid','Cid',149,0x0000E10B00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,349,0,0,1052,149);
 INSERT INTO `mob_pools` VALUES (5938,'gilgamesh','Gilgamesh',149,0x0000E20B00000000000000000000000000000000,12,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,350,0,0,1053,149);
 INSERT INTO `mob_pools` VALUES (5939,'areuhat','Areuhat',145,0x0000E30B00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,351,0,0,1054,145);
 INSERT INTO `mob_pools` VALUES (5940,'semih_lafihna','SemihLafihna',151,0x0000E40B00000000000000000000000000000000,11,0,25,500,200,3,0,0,0,0,0,32,0,3,0,0,352,0,0,1055,151);
 INSERT INTO `mob_pools` VALUES (5941,'elivira','Elivira',149,0x0000E50B00000000000000000000000000000000,11,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,353,0,0,1056,149);
 INSERT INTO `mob_pools` VALUES (5942,'noillurie','Noillurie',145,0x0000E60B00000000000000000000000000000000,12,7,3,240,100,0,0,0,0,0,0,32,0,3,0,0,354,0,0,1057,145);
 INSERT INTO `mob_pools` VALUES (5943,'lhu_mhakaracca','LhuMhakaracca',151,0x0000E70B00000000000000000000000000000000,9,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,355,0,0,1058,151);
-INSERT INTO `mob_pools` VALUES (5944,'ferreous_coffin','FerreousCoffin',146,0x0000E80B00000000000000000000000000000000,3,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,356,0,0,1059,146);
+INSERT INTO `mob_pools` VALUES (5944,'ferreous_coffin','FerreousCoffin',146,0x0000E80B00000000000000000000000000000000,3,1,3,240,100,3,0,0,0,0,0,32,0,3,0,0,356,0,0,1059,146);
 INSERT INTO `mob_pools` VALUES (5945,'lilisette','Lilisette',484,0x0000E90B00000000000000000000000000000000,19,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,357,0,0,1060,484);
 INSERT INTO `mob_pools` VALUES (5946,'mumor','Mumor',149,0x0000EA0B00000000000000000000000000000000,19,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,358,0,0,1061,149);
 INSERT INTO `mob_pools` VALUES (5947,'uka_totlihn','UkaTotlihn',151,0x0000EB0B00000000000000000000000000000000,19,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,359,0,0,1062,151);
 INSERT INTO `mob_pools` VALUES (5948,'klara','Klara',149,0x0000ED0B00000000000000000000000000000000,1,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,360,0,0,1063,149);
 INSERT INTO `mob_pools` VALUES (5949,'romaa_mihgo','RomaaMihgo',151,0x0000EE0B00000000000000000000000000000000,6,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,361,0,0,1064,151);
 INSERT INTO `mob_pools` VALUES (5950,'kuyin_hathdenna','KuyinHathdenna',151,0x0000EF0B00000000000000000000000000000000,21,0,3,240,100,0,0,0,0,0,0,32,0,3,0,0,362,0,0,1065,151);
 INSERT INTO `mob_pools` VALUES (5951,'rahal','Rahal',145,0x0000F00B00000000000000000000000000000000,7,1,3,240,100,0,0,0,0,0,0,32,0,3,0,0,363,0,0,1066,145);
-INSERT INTO `mob_pools` VALUES (5952,'koru-moru','Koru-Moru',153,0x0000F10B00000000000000000000000000000000,5,0,3,240,100,1,0,0,0,0,0,32,0,3,0,0,364,0,0,1067,153);
+INSERT INTO `mob_pools` VALUES (5952,'koru-moru','Koru-Moru',153,0x0000F10B00000000000000000000000000000000,5,0,3,240,100,3,0,0,0,0,0,32,0,3,0,0,364,0,0,1067,153);
```

Comment:
Koru is def not correct as long range. he gets in my way in melee range all the time on retail. I wish he was long range.

Reply:
according to wiki the deal is that he just does not move at all. same for Kupipi, which is why she'll melee if summoned early in the lineup - because she'll already be closer.

> Will remain stationary wherever he is standing upon engaging an enemy. Will not attempt to move closer to the enemy unless it moves out of his casting range.

Koru is in my way all the time because he's a non engaging statue of a caster, and you might have managed to start battles with him farther out.